src/main/java/tictactoe/ComputerPlayer.java

```diff
@@ -53,8 +53,8 @@ private boolean couldEvaluateMove(int bestMoveEvaluated) {
     }
 
     private int randomMove() {
-        int index = new Random().nextInt(board.freePosition().size());
-        return board.freePosition().get(index);
+        int index = new Random().nextInt(board.freePositions().size());
```

Comment:
You could add a method to the board `board.getNumberOfFreePositions()` which returns the size. This reduces the coupling between the `ComputerPlayer` and the 'Board' as if you changed the underlying implementation of the `Board` such that the size was obtained in a different way, only the `Board` would change. Whereas at the moment your `ComputerPlayer` would also have to change. (Tell don't ask principle).